tools/apiview/parsers/cpp-api-parser/ApiViewProcessor/AstNode.cpp

```diff
@@ -1985,6 +2130,10 @@ class AstConstructor : public AstMethod {
   }
   void DumpNode(AstDumper* dumper, DumpNodeOptions const& dumpOptions) const override
   {
+    if (dumpOptions.NeedsDocumentation)
+    {
+      DumpDocumentation(dumper, dumpOptions);
+    }
```

Comment:
Could simplify to avoid the if check by calling `DumpDocumentation(dumper, dumpOptions);` and letting that method do nothing if the dumpOptions.NeedsDocumentation == false.  Saves writing the if check everywhere

Reply:
Great suggestion.